phpunit.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hi @Th3Mouk , did you use [EditorConfig](http://editorconfig.org/#example-file) to solve  the indent prpblem?